instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/extensions/bedrock.py

```diff
@@ -140,6 +218,111 @@ def on_success(self, span: Span, result: dict[str, Any]):
                 [stop_reason],
             )
 
+    def _invoke_model_on_success(
+        self, span: Span, result: dict[str, Any], model_id: str
+    ):
+        original_body = None
+        try:
+            original_body = result["body"]
+            body_content = original_body.read()
+
+            # Use one stream for telemetry
+            stream = io.BytesIO(body_content)
+            telemetry_content = stream.read()
+            response_body = json.loads(telemetry_content.decode("utf-8"))
```

Comment:
Can you just pass `body_content` directly since it's already buffered?
```suggestion
            response_body = json.loads(body_content)
```

Reply:
Good catch, removed the double copy :)